src/main/scala/exercises/06_Collections.scala

```diff
@@ -102,9 +90,7 @@ object Collections {
    * Again, easy to implement using built-in functionality, but also possible 
    * to implement in your own free-style way.
    */
-  def elementExists[T](l: List[T], e: T): Boolean = {
-    error("fix me")
-  }
+  def elementExists[T](l: List[T], e: T): Boolean = l.contains(e)
```

Comment:
I think `l.contains(e)` is one of the function calls that is more readable as `l contains e`, which is not necessarily true for all functions like `filter`.